packages/client/src/client/components/TabBar/TabBar.tsx

```diff
@@ -75,7 +78,7 @@ export const TabBar = <T extends string>({
             </Action>
           ))}
       {handleItemOnClick && !doNotShowDropdown && (
-        <DropdownWrapper>
+        <DropdownWrapper data-testid="nav-dropdown">
```

Comment:
this test ID is too vague .. needs to be `rfq-filter-dropdown` or similar .. and like I was suggesting, we can start to use Locators, per alpaca-reference-ui .. plenty of refactoring to do around E2E (you should be happy, refactoring is fun 😁)